build-logic/src/main/kotlin/com/adevinta/spark/SparkAndroidComposePlugin.kt

```diff
@@ -39,6 +41,14 @@ internal class SparkAndroidComposePlugin : Plugin<Project> {
                 }
             }
 
+            @Suppress("ktlint:max-line-length")
```

Comment:
Didn't we had this already 🤔 ?

Reply:
The `ktlint` suppression or the `suppressKotlinVersionCompatibilityCheck`?

Reply:
this is not needed when the kotlin version matches the expected version requested by the compose compiler. And this is not the case here.
Compose compiler version 1.4.7 expects Kotlin 1.8.21.
This is not very important for such patch release, but we can wait until compose releases (if it ever does) a new 1.4.8 version.

Reply:
Okay good to know thanks